build_tools/packaging/linux/package.json

```
@@ -0,0 +1,184 @@
+
+[
+ {
+   "Package" : "rocm-core",
+   "Version" : "7.1.0",
+   "Architecture" : "amd64",
+   "BuildArch" : "x86_64",
+   "DEBDepends" : [ "python3", "libc6"],
+   "RPMRequires" : [ "python3", "glibc"],
+   "Maintainer" : "ROCm Dev Support <rocm-dev.support@amd.com>",
+   "Description" : "ROCM-CORE package helps to get ROCm release version and install path",
+   "Section" : "devel",
+   "Priority" : "optional",
+   "Group" : "unknown",
+   "License" : "MIT",
+   "Vendor" : "Advanced Micro Devices, Inc",
+   "Homepage" : "https://github.com/AMD-ROCm-Internal/rocm-core",
```

public URL: https://github.com/ROCm/rocm-core, recently moved to https://github.com/ROCm/rocm-systems

```suggestion
   "Homepage" : "https://github.com/ROCm/rocm-systems/tree/develop/projects/rocm-core",
```

```suggestion
   "Homepage" : "https://github.com/ROCm/rocm-systems",
```